### PR TITLE
[#939] Apps link now open in new tab.

### PR DESCRIPTION
**Pull request by mauriel133**
Also removed unused reference in base html

### CODE REVIEW

File: hs_core/templates/400.html [hunk 1]
```diff
@@ -292,7 +292,5 @@ <h1 class="animated slideInLeft"><span>{% block title %}{% endblock %}</span></h
         </div>
     </div>
 
-    {% include "includes/footer_scripts.html" %}
-
```
Comment (mauriel133) [inline hs_core/templates/400.html]: This referenced file does not exist anymore. I'm assuming it was deleted at some point, but this reference was forgotten. Can someone confirm?

**Comment by hydrobuild** [comment]
Can one of the admins verify this patch?

**Comment by pkdash** [comment]
@hyi Can you review this small PR?

**Comment by mjstealey** [comment]
verified working on riverdev

**Pull request merged**